src/main/kotlin/dev/mfazio/wwc2023/types/domain/Team.kt

```diff
@@ -7,73 +7,73 @@ sealed class Team(
     val group: String,
     val altIds: List<String> = listOf(teamId)
 ) {
-    object Argentina : Team("ARG", "Argentina", "AR", "C")
-    object Australia : Team("AUS", "Australia", "AU", "D")
-    object Belgium : Team("BEL", "Belgium", "BE", "F")
-    object Brazil : Team("BRA", "Brazil", "BR", "G")
-    object Cameroon : Team("CMR", "Cameroon", "CM", "G")
-    object Canada : Team("CAN", "Canada", "CA", "F")
-    object CostaRica : Team("CRC", "Costa Rica", "CR", "E")
-    object Croatia : Team("CRO", "Croatia", "HR", "F")
+    object Argentina : Team("ARG", "Argentina", "AR", "G")
+    object Australia : Team("AUS", "Australia", "AU", "B")
+    object Brazil : Team("BRA", "Brazil", "BR", "F")
+    object Canada : Team("CAN", "Canada", "CA", "B")
+    object ChinaPR : Team("CHN", "China PR", "CN", "D")
```

Comment:
We can just make this `China`. It's easier, plus the "People's Republic" part is a sham anyway 🤷🏻 
```suggestion
    object China : Team("CHN", "China", "CN", "D")
```